TundraTD/Assets/Scripts/City/Building/Upgrades/FireTowerUpgrades/MeteorLandingImpulsUpgrade.cs

```diff
@@ -0,0 +1,20 @@
+using City.Building.ElementPools;
+using Spells;
+using UnityEngine;
+
+namespace City.Building.Upgrades.FireTowerUpgrades
+{
+    public class MeteorLandingImpulsUpgrade : IUpgrade
```

Comment:
A typo in the class name, fix it
```suggestion
    public class MeteorLandingImpulseUpgrade : IUpgrade
```